common/src/main/webapp/sass/explore.scss

```diff
@@ -65,7 +65,7 @@ body.hide-reusability {
   line-height: var(--pl-input-line-height);
   border: 1px dashed transparent; // the border makes the sizing work.
   /* stylelint-disable-next-line csstools/value-no-unknown-custom-properties */
-  padding: var(--pl-input-padding)
+  padding: var(--pl-input-padding);
```

Comment:
Did you run this through a formatter to get these?
We may want to agree if so, I remember using prettier at some moment but we haven't enforced it

Reply:
Yeah my editor auto-formatted it with Prettier. I can revert it if you want, but since it also added semicolons (which _should_ be in valid css anyway) I decided to commit it. I can revert it if you want

Reply:
I think it is fine, maybe others can chime in but we may want to have the same settings or we could endup changing the format back and forth 😄